vnode.go

```diff
@@ -19,14 +20,6 @@ type vnode struct {
 	node dom.Node
 }
 
-func init() {
```

Comment:
I'm also not a fan of `init` functions in general. The idea behind this was to know immediately if the browser is not available, is this behavior captured anywhere now?